### PR TITLE
loader: remove ULL suffix from assembly file

### PR DESCRIPTION
**Pull request by YadongQi**
Some old version of GCC/GNU AS does not support "ULL"
suffix in assemble file.

Signed-off-by: Qi Yadong <yadong.qi@intel.com>

### CODE REVIEW

File: loader/stage0/abl/stage0_entry.S [hunk 1]
```diff
@@ -224,9 +224,9 @@ bspstack:
 
 .balign 8
 gdt_x64:
-	.quad 	0x0000000000000000ULL	/* NULL descriptor */
```
Comment (kwang13) [inline loader/stage0/abl/stage0_entry.S]: in the commit message, tell more about why this change is required.

Comment (YadongQi) [reply loader/stage0/abl/stage0_entry.S]: add unsupported GCC version in commit message?

Comment (YadongQi) [reply loader/stage0/abl/stage0_entry.S]: these info already in commit message.
